content/docs/plugins.md

```diff
@@ -132,7 +132,8 @@ Args:
 Return: none  
 Description:
 	This function should return the description to be displayed in the first field of the status line.  
-	The parameters are to be treated in the same way as in the GetLexerName function.  
+	The parameters are to be treated in the same way as in the GetLexerName function, except for 
+	the name parameter, here a utf16 encoded name is expected.
```

Comment:
maybe change "here a utf16 encoded name is expected" to "which requires a string encoded with UTF-16."